HousingSearchListener/V1/Interfaces/ElasticSearchService.cs

```diff
@@ -53,21 +53,33 @@ private async Task<HttpResponseMessage> GetPersonFromPersonApi(SQSEvent.SQSMessa
             return result;
         }
 
-        private async Task UpdateEsIndexWithCreatedPerson(HttpResponseMessage result)
+        private async Task UpdateEsIndexWithCreatedPerson(SQSEvent.SQSMessage record, HttpResponseMessage result)
         {
+            var personCreatedMessage = _personMessageFactory.Create(record);
+
             var personString = await result.Content.ReadAsStringAsync();
             var person = JsonConvert.DeserializeObject<Person>(personString);
 
             var esPerson = _esPersonFactory.Create(person);
 
-            await _esHelper.Create(esPerson);
+            switch (personCreatedMessage.EventType)
+            {
+                case EventTypes.PersonCreatedEvent:
+                    await _esHelper.Create(esPerson);
+                    break;
+                case EventTypes.PersonUpdatedEvent:
+                    await _esHelper.Update(esPerson);
+                    break;
+                default:
+                    await _esHelper.Create(esPerson);
```

Comment:
I think the default should be to do nothing...

Reply:
I would've set it to default, I am just worried about potentially not setting the eventtype on the message in one of the calling apps.